packages/ui/components/form/timezone-select/TimezoneSelect.tsx

```diff
@@ -52,18 +52,22 @@ export function TimezoneSelect({
       classNames={{
         ...timezoneClassNames,
         input: (state) =>
-          classNames("text-emphasis h-6", timezoneClassNames?.input && timezoneClassNames.input(state)),
+          classNames(
+            "text-emphasis h-6 md:max-w-[145px] max-w-[250px]",
+            timezoneClassNames?.input && timezoneClassNames.input(state)
+          ),
         option: (state) =>
           classNames(
-            "bg-default flex cursor-pointer justify-between py-2.5 px-3 rounded-none text-default ",
+            "bg-default flex !cursor-pointer justify-between py-2.5 px-3 rounded-none text-default ",
             state.isFocused && "bg-subtle",
-            state.isSelected && "bg-emphasis text-default",
+            state.isSelected && "bg-emphasis",
             timezoneClassNames?.option && timezoneClassNames.option(state)
           ),
         placeholder: (state) => classNames("text-muted", state.isFocused && "hidden"),
         dropdownIndicator: () => "text-default",
         control: (state) =>
           classNames(
+            "!cursor-pointer",
```

Comment:
What does the bang `!` does in this context?

Reply:
! just adds important to cursor-pointer. Changes are react-select doesnt remove this from their core styles so we need to force it